model_compression_toolkit/core/pytorch/reader/graph_builders.py

```diff
@@ -52,6 +52,24 @@ def forward(self):
         return getattr(self, CONSTANT)
 
 
+class BufferHolder(torch.nn.Module):
```

Comment:
Maybe we need to move these classes to a different file? "node_holders" or something of the sort? @haihabi what do you think?

Reply:
Agree